packages/react-ui-core/src/Counter/__tests__/Counter-test.js

```diff
@@ -17,10 +17,9 @@ const onClick = (type, counter) => {
 
 describe('Counter', () => {
   it('renders a Counter', () => {
-    const wrapper = shallow(<Counter theme={theme} />)
-    expect(wrapper.find('span[role="decrementUnit"]')).toHaveLength(1)
+    const wrapper = shallow(<Counter id="foo" theme={theme} />)
+    expect(wrapper.find('span[role="decrementUnit"]')).toHaveLength(0)
```

Comment:
What is the point of this? You're saying it doesn't have a decrement counter???? You should probably check the name of the referenced selector here.  Maybe using an `id` is more appropriate.